diff -aurN node-v6.6.0.org/deps/v8_inspector/third_party/v8_inspector/platform/v8_inspector/v8_inspector.gyp node-v6.6.0/deps/v8_inspector/third_party/v8_inspector/platform/v8_inspector/v8_inspector.gyp
--- node-v6.6.0.org/deps/v8_inspector/third_party/v8_inspector/platform/v8_inspector/v8_inspector.gyp	2016-09-14 17:37:12.000000000 +0000
+++ node-v6.6.0/deps/v8_inspector/third_party/v8_inspector/platform/v8_inspector/v8_inspector.gyp	2016-09-16 05:09:03.421459890 +0000
@@ -71,7 +71,7 @@
                   '<(blink_platform_output_dir)/v8_inspector/public/protocol/Schema.h',
                 ],
                 'action': [
-                  'python',
+                  'python2.7',
                   '../inspector_protocol/CodeGenerator.py',
                   '--output_base', '<(blink_platform_output_dir)',
                   '--config', 'inspector_protocol_config.json',
@@ -99,7 +99,7 @@
           ],
           'outputs': [ '<(SHARED_INTERMEDIATE_DIR)/blink/platform/v8_inspector/InjectedScriptSource.h', ],
           'action': [
-            'python', 'build/xxd.py', 'InjectedScriptSource_js', 'InjectedScriptSource.js', '<@(_outputs)'
+            'python2.7', 'build/xxd.py', 'InjectedScriptSource_js', 'InjectedScriptSource.js', '<@(_outputs)'
           ],
         },
       ],
@@ -119,7 +119,7 @@
           ],
           'outputs': [ '<(SHARED_INTERMEDIATE_DIR)/blink/platform/v8_inspector/DebuggerScript.h', ],
           'action': [
-            'python', 'build/xxd.py', 'DebuggerScript_js', 'DebuggerScript.js', '<@(_outputs)'
+            'python2.7', 'build/xxd.py', 'DebuggerScript_js', 'DebuggerScript.js', '<@(_outputs)'
           ],
         },
       ],
@@ -141,7 +141,7 @@
             '<(blink_platform_output_dir)/v8_inspector/protocol.json',
           ],
           'action': [
-            'python',
+            'python2.7',
             '../inspector_protocol/generate-inspector-protocol-version',
             '--o',
             '<@(_outputs)',
@@ -211,7 +211,7 @@
             '<(blink_platform_output_dir)/v8_inspector/public/protocol/Schema.h',
           ],
           'action': [
-            'python',
+            'python2.7',
             '../inspector_protocol/CodeGenerator.py',
             '--output_base', '<(blink_platform_output_dir)',
             '--config', 'inspector_protocol_config_stl.json',
