diff -aurN node-v5.2.0.org/Makefile node-v5.2.0/Makefile
--- node-v5.2.0.org/Makefile	2015-12-14 06:44:39.713806143 +0000
+++ node-v5.2.0/Makefile	2015-12-14 06:45:19.215340677 +0000
@@ -20,9 +20,8 @@
 NODE_G_EXE = node_g$(EXEEXT)
 
 # Flags for packaging.
-BUILD_DOWNLOAD_FLAGS ?= --download=all
-BUILD_INTL_FLAGS ?= --with-intl=small-icu
-BUILD_RELEASE_FLAGS ?= $(BUILD_DOWNLOAD_FLAGS) $(BUILD_INTL_FLAGS)
+BUILD_INTL_FLAGS ?= --with-intl=system-icu
+BUILD_RELEASE_FLAGS ?= $(BUILD_INTL_FLAGS)
 
 # Default to verbose builds.
 # To do quiet/pretty builds, run `make V=` to set V to an empty string,
