diff -aurN node-v6.8.0.org/Makefile node-v6.8.0/Makefile
--- node-v6.8.0.org/Makefile	2016-10-13 01:20:28.446390268 +0000
+++ node-v6.8.0/Makefile	2016-10-13 01:23:54.308382678 +0000
@@ -42,9 +42,8 @@
 NPM ?= ./deps/npm/bin/npm-cli.js
 
 # Flags for packaging.
-BUILD_DOWNLOAD_FLAGS ?= --download=all
-BUILD_INTL_FLAGS ?= --with-intl=small-icu
-BUILD_RELEASE_FLAGS ?= $(BUILD_DOWNLOAD_FLAGS) $(BUILD_INTL_FLAGS)
+BUILD_INTL_FLAGS ?= --with-intl=system-icu
+BUILD_RELEASE_FLAGS ?= $(BUILD_INTL_FLAGS)
 
 # Default to verbose builds.
 # To do quiet/pretty builds, run `make V=` to set V to an empty string,
