diff -aurN node-v4.6.2.org/Makefile node-v4.6.2/Makefile
--- node-v4.6.2.org/Makefile	2016-11-09 03:54:01.774583282 +0000
+++ node-v4.6.2/Makefile	2016-11-09 03:56:29.802577824 +0000
@@ -38,9 +38,8 @@
 NPM ?= ./deps/npm/bin/npm-cli.js
 
 # Flags for packaging.
-BUILD_DOWNLOAD_FLAGS ?= --download=all
-BUILD_INTL_FLAGS ?= --with-intl=small-icu
-BUILD_RELEASE_FLAGS ?= $(BUILD_DOWNLOAD_FLAGS) $(BUILD_INTL_FLAGS)
+BUILD_INTL_FLAGS ?= --with-intl=system-icu
+BUILD_RELEASE_FLAGS ?= $(BUILD_INTL_FLAGS)
 
 # Default to verbose builds.
 # To do quiet/pretty builds, run `make V=` to set V to an empty string,
