diff -aurN node-v6.4.0.orig/deps/v8_inspector/third_party/v8_inspector/platform/v8_inspector/v8_inspector.gyp node-v6.4.0/deps/v8_inspector/third_party/v8_inspector/platform/v8_inspector/v8_inspector.gyp
--- node-v6.4.0.orig/deps/v8_inspector/third_party/v8_inspector/platform/v8_inspector/v8_inspector.gyp  2016-08-15 14:27:14.000000000 +0000
+++ node-v6.4.0/deps/v8_inspector/third_party/v8_inspector/platform/v8_inspector/v8_inspector.gyp       2016-08-17 03:06:03.691139553 +0000
@@ -20,7 +20,7 @@
           ],
           'outputs': [ '<(SHARED_INTERMEDIATE_DIR)/blink/platform/v8_inspector/InjectedScriptSource.h', ],
           'action': [
-            'python', 'build/xxd.py', 'InjectedScriptSource_js', 'InjectedScriptSource.js', '<@(_outputs)'
+            'python2.7', 'build/xxd.py', 'InjectedScriptSource_js', 'InjectedScriptSource.js', '<@(_outputs)'
           ],
         },
       ],
@@ -40,7 +40,7 @@
           ],
           'outputs': [ '<(SHARED_INTERMEDIATE_DIR)/blink/platform/v8_inspector/DebuggerScript.h', ],
           'action': [
-            'python', 'build/xxd.py', 'DebuggerScript_js', 'DebuggerScript.js', '<@(_outputs)'
+            'python2.7', 'build/xxd.py', 'DebuggerScript_js', 'DebuggerScript.js', '<@(_outputs)'
           ],
         },
       ],
@@ -74,7 +74,7 @@
           'action_name': 'generateV8InspectorProtocolBackendSources',
           'inputs': [
             '<@(jinja_module_files)',
-            # The python script in action below.
+            # The python2.7 script in action below.
             '../inspector_protocol/CodeGenerator.py',
             # Source code templates.
             '../inspector_protocol/TypeBuilder_h.template',
@@ -99,7 +99,7 @@
             '<(blink_platform_output_dir)/v8_inspector/public/protocol/Debugger.h',
           ],
           'action': [
-            'python',
+            'python2.7',
             '../inspector_protocol/CodeGenerator.py',
             '--protocol', 'js_protocol.json',
             '--string_type', 'String16',
@@ -128,7 +128,7 @@
             '<(blink_platform_output_dir)/v8_inspector/protocol.json',
           ],
           'action': [
-            'python',
+            'python2.7',
             '../inspector_protocol/generate-inspector-protocol-version',
             '--o',
             '<@(_outputs)',
